lib/tasks/ci.rake

```diff
@@ -0,0 +1,9 @@
+unless Rails.env.production?
```

Comment:
important that you make the coveralls setup a separate PR in the future!

Reply:
Confirmed